### PR TITLE
Requests integration frontend

### PR DESCRIPTION
**Pull request by kramstrom**

### CODE REVIEW

File: backend/middleware/.env [hunk 1]
```diff
@@ -1,8 +1,8 @@
 SERVER_HOST=localhost
-SERVER_PORT=3000
+SERVER_PORT=3001
 
 WS_HOST = localhost
-WS_PORT = 3020
+WS_PORT = 5001
 
 MOCK_HOST = localhost
```
Comment (falapen) [inline backend/middleware/.env]: Remove Unused port/host

File: frontend/src/components/DrawerNewsList.vue [hunk 2]
```diff
@@ -1,56 +1,60 @@
...
 </template>
 
 <script>
 import NewsList from './NewsList'
-import { mapGetters, mapActions } from 'vuex';
+import { mapGetters, mapActions } from 'vuex'
 
 export default {
-  name: 'drawernewslist',
+  name: 'Drawernewslist',
```
Comment (falapen) [inline frontend/src/components/DrawerNewsList.vue]: DrawerNewsList ???

Comment (kramstrom) [reply frontend/src/components/DrawerNewsList.vue]: Maybe

File: backend/middleware/.env (hunk 1 above)
Comment (kramstrom) [reply backend/middleware/.env]: Maybe

File: frontend/src/components/NotificationsMunicipality.vue [hunk 3]
```diff
@@ -2,160 +2,166 @@
...
 
       <!--TEXT-->
-      <transition
-        v-on:beforeEnter="(el, done) => textBeforeEnter(el, done, municipality)"
-        v-on:enter="(el, done) => textEnter(el, done, municipality)"
-        v-on:leave="(el, done) => textLeave(el, done, municipality)"
-        v-bind:css="false">
+      <Transition
+        :css="false"
+        @beforeEnter="(el, done) => textBeforeEnter(el, done, municipality)"
```
Comment (falapen) [inline frontend/src/components/NotificationsMunicipality.vue]: Variable-name "el" == "element" ?

File: frontend/src/components/NotificationsMunicipality.vue [hunk 4]
```diff
@@ -2,160 +2,166 @@
...
+import { mapGetters, mapActions } from 'vuex'
+import Velocity from 'velocity-animate'
 
 export default {
-  name: "notificationsMunicipality",
+  name: 'NotificationsMunicipality',
   props: ['circleSize', 'fontSize', 'yOffset', 'calculateNewsLengthForObjects', 'updateNewsLengthForObjects', 'lineWidth'],
+  data: function () {
+    return {
+      previousmunicipalityNewsLength: []
```
Comment (falapen) [inline frontend/src/components/NotificationsMunicipality.vue]: incorrect case -> previousMunicipalityNewsLength

**Pull request merged**